### DISCUSSION TITLE
Strange #<Issue:…> markup in some notifications

### DISCUSSION THREAD

**Discussion by KalleOlaviNiemitalo** [community/community#8490] [Mobile] [answered]
Some notifications are showing markup like `#<Issue:0x00007f952e10b510>` instead of the issue description. Not all notifications, though. Using GitHub 1.42.0 app on iOS 15.1.

![image](https://user-images.githubusercontent.com/46201428/144987007-75bfad32-d488-4521-a5c1-e2107f52861f.png)

**Comment by canuckjacq** [comment, marked as answer]
Hi, and thank for the report! While we won't be able to do anything about the existing notifications with this, newer ones should display correctly from now on, even in existing app releases.
